packages/esm-outpatient-app/src/past-visit/past-visit.scss

```
@@ -0,0 +1,99 @@
+@import "~@openmrs/esm-styleguide/src/vars";
+@import "~carbon-components/src/globals/scss/vars";
+@import "~carbon-components/src/globals/scss/mixins";
+
+
+.bodyShort02 {
+  @include carbon--type-style("body-short-02");
+}
+
+.bodyLong01 {
+  @include carbon--type-style("body-long-01");
+}
+
+.date {
+  @include carbon--type-style('label-01');
+  color: $text-02;
+}
+
+.visitType {
+  @include carbon--type-style('productive-heading-02');
+  color: $text-01;
+}
+
+.container {
+  background-color: $ui-background;
+  border: 1px solid $grey-2;
+  padding: 1rem;
+  margin: 1rem 0rem 1rem;
```

Would be preferable to use carbon spacing
```suggestion
  margin: $spacing-05 0rem $spacing-05;
```